packages.json

```diff
@@ -23156,5 +23156,22 @@
     "description": "A Nim client for Todoist's REST API",
     "license": "Apache-2.0",
     "web": "https://ruivieira.github.io/nim-todoist/index.html"
+  },
+  {
+    "name": "pl0t",
+    "url": "https://github.com/al6x/pl0t?subdir=api/nim",
```

Comment:
since this is a pl0t repo already, why the subdir?

Reply:
It's monorepo, there are couple of different things, the language APIs reside in `/api` folder, so there going to be `/api/nim` and `/api/typescript`.